Source/embeddings/Builder/EmbeddingAlreadyHasARemoveDecorator.ts

```diff
@@ -5,7 +5,7 @@ import { Exception } from '@dolittle/rudiments';
 import { Constructor } from '@dolittle/types';
 
 /**
- * Exception that gets thrown when attempting to register more than one @remove() decorators on an embedding.
+ * Exception that gets thrown when attempting to register more than one @resolveDeletionToEvents() decorators on an embedding.
  */
 export class EmbeddingAlreadyHasARemoveDecorator extends Exception {
```

Comment:
```suggestion
export class EmbeddingAlreadyHasADeletionDecorator extends Exception {
```
Let's also rename all of these specific exceptions

Reply:
Done.